packages/slate-plugins/src/elements/code-block/components/EditorChild.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+import { RenderElementProps } from 'slate-react';
+
+//
+
+export const EditorChild = ({ attributes, children }: RenderElementProps) => {
```

Comment:
Next step, you can use using Slate.onChange, Slate.value and EditablePlugins.plugins = [CodeBlockPlugin] like a storybook example (we'll generalize the options later)